about.html

```diff
@@ -71,7 +71,278 @@
+                <div class="main__principles-box">
+                  <img src="./assets/images/about/emp-icon.svg" alt="icon" />
+                  <h3 class="main__principles-box-title">
+                    Клиентоориентированность
+                  </h3>
+                  <span class="main__principles-box-text"
+                    >Мы понимаем как тяжело начинать как делать помогаем
+                    новичкам
+                  </span>
+                </div>
+                <div class="main__principles-box">
+                  <img src="./assets/images/about/lamp-icon.svg" alt="icon" />
+                  <h3 class="main__principles-box-title">
+                    Предоставление качественных услуг
+                  </h3>
+                  <span class="main__principles-box-text"
+                    >Мы понимаем как тяжело начинать как делать помогаем
+                    новичкам
+                  </span>
+                </div>
+                <div class="main__principles-box">
+                  <img
+                    src="./assets/images/about/business-icon.svg"
+                    alt="icon"
+                  />
+                  <h3 class="main__principles-box-title">Развитие бизнеса</h3>
+                  <span class="main__principles-box-text"
+                    >Мы понимаем как тяжело начинать как делать помогаем
+                    новичкам
+                  </span>
+                </div>
+                <div class="main__principles-box">
+                  <img src="./assets/images/about/eye-icon.svg" alt="icon" />
+                  <h3 class="main__principles-box-title">Конфиденциальность</h3>
+                  <span class="main__principles-box-text"
+                    >Мы понимаем как тяжело начинать как делать помогаем
+                    новичкам
+                  </span>
+                </div>
+                <div class="main__principles-box">
+                  <img src="./assets/images/about/loupe-icon.svg" alt="icon" />
+                  <h3 class="main__principles-box-title">Грамотный подход</h3>
+                  <span class="main__principles-box-text"
+                    >Мы понимаем как тяжело начинать как делать помогаем
+                    новичкам
+                  </span>
+                </div>
+                <div class="main__principles-box">
+                  <img
+                    src="./assets/images/about/calendar-icon.svg"
+                    alt="icon"
+                  />
+                  <h3 class="main__principles-box-title">Соблюдение сроков</h3>
+                  <span class="main__principles-box-text"
+                    >Мы понимаем как тяжело начинать как делать помогаем
+                    новичкам
+                  </span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="main__wr-certificates">
+          <div class="main__container">
```

Comment:
Я так понимаю, ты используешь этот элемент контейнера main__container в нескольких разделах страницы. Ну, если это так, то наверное он должен быть блоком. Если вдруг понадобятся отступы от контейнера, то можно сделать модификатор или микс для него. Ну т.е., учитывая, что ты повторяешь этот контейнер, наверное, его стоило сделать блоком.
```suggestion
          <div class="container">
```